### PR TITLE
fix: use timestamp with timezone for signing_keys

### PR DESCRIPTION
**Pull request by erikxiv**
Signed-off-by: Erik Larsson <erik.larsson@schibsted.com>

## Hey, I just made a Pull Request!

Fixes https://github.com/backstage/backstage/issues/5058

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
Double check that sqlite supports this alter. https://sqlite.org/lang_altertable.html You'll see a lot of other migrations in the repo check `if (knex.client.config.client !== 'sqlite3') {` around these types of changes since its support for alterations is limited.

**Review by benjdlambert — APPROVED** [review]

File: .changeset/hungry-jars-knock.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```
Comment (benjdlambert) [inline .changeset/hungry-jars-knock.md:2]: Think this is a patch release?

**Comment by benjdlambert** [comment]
Good to go apart from the comment about patch changeset instead of minor. Whilst we are <1.0.0 minor === patch.

**Comment by benjdlambert** [comment]
@erikxiv can you rebase again, we had an unstable master 😱

**Comment by erikxiv** [comment]
Rebased

**Pull request merged**